examples/rivets/index.html

```diff
@@ -0,0 +1,123 @@
+<!doctype html>
+<html lang="en" data-framework="rivets">
+	<head>
+		<meta charset="utf-8">
+		<title>Rivets • TodoMVC</title>
+		<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
+		<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+	</head>
+	<body>
+		<script type="text/template" id="todos">
+			<header id="header">
```

Comment:
@samccone can you please reassure me that i need to use classes and css@2.0.
You wont got any test coverage - all the tests fail.